### DISCUSSION TITLE
Populate the username with date and time when a record was updated

### DISCUSSION THREAD

**Discussion by crawfishboy** [statamic/cms#5658] [General]
Does anyone have a suggestion on how to log the username, date, and time a record was updated?

**Comment by duncanmcclean** [comment]
Out of the box, this data is actually already populated. If you look at an entry's markdown file:

* `updated_by` is the ID of the user who updated the entry
* `updated_at` is a timestamp of when the entry was updated

**Reply by crawfishboy** [reply]
Thanks, but I don't see this information in the markdown files.  Is there something I need to do to implement it and how can I get this information to display in the control panel and not on the front end?

**Reply by jasonvarga** [reply]
Make sure this is `true` https://github.com/statamic/statamic/blob/1733fea5ff4855d72ca2a163d155f9a23d6f833a/config/statamic/system.php#L84

**Comment by crawfishboy** [comment]
We do not have the statamic directory under statamic/config.  Is this something that needs to be added?  I really appreciate your input. 
 Here is a view of our file structure:

![image](https://user-images.githubusercontent.com/74839338/160243154-07e6ef92-794c-4130-b131-9a1f4ef67d8b.png)

**Reply by duncanmcclean** [reply]
Ohh. Are you using Statamic v2?

**Reply by crawfishboy** [reply]
Yes, Statamic v2.
